src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="search" 
+      action="/search" 
+      method="post" 
+      data-qa="keypress"
+    >
+      <img 
+        src="./images/Search.svg" 
+        alt="pic" 
+        class="search-icon"
+      >
+      <label>
+        <input 
+          data-qa="big"
+          type="search" 
+          class="queryy query" 
+          name="q" 
```

Comment:
just use one of those classes
give a proper value for the name attribute, not just q